masonite/auth/Auth.py

```diff
@@ -63,9 +63,23 @@ def login(self, name, password):
         """
         auth_column = self.auth_model.__auth__
         try:
-            model = self.auth_model.where(auth_column, name).first()
+            password_column = self.auth_model.password if hasattr(self.auth_model, 'password') else self.auth_model.__password__
+        except AttributeError as e:
+            raise AttributeError('Your model does not have a password column or a designated __password__ attribute. Set the __password__ attribute to the name of your password column.') from e
 
-            if model and bcrypt.checkpw(bytes(password, 'utf-8'), bytes(model.password, 'utf-8')):
+        try:
+            # Try to login multiple or statements if given an auth list
+            if isinstance(auth_column, list):
```

Comment:
so the list can more than 2 fields to authenticate the user?

['username', 'email', 'mob_number']

Reply:
yes. It will take the first one and do a `where`, and then do a `or_where` for each other element in the list.

Reply:
wow, this is good feature. LGTM